### DISCUSSION TITLE
Support for GriefDefender claims

### DISCUSSION THREAD

**Discussion by catronomix** [maruohon/minihud#326] [Ideas]
Visualizing claim borders from the GriefDefender spigot plugin (and other server claim plugins) would be an awesome addition to this amazing mod!

Of course this will depend on getting those borders in the client, which might require an add-on server plugin to be created...
Just an idea ^^

**Comment by maruohon** [comment]
I could think about adding some kind of a "more generic (box?) renderer" that can receive data from a server-side mod or plugin. Then it could be used for many different things without really caring about what it's being used for.

I'm not interested in developing any Bukkit plugins though, so someone else would need to make the plugin to send for example the GriefDefender data.
